# msg/TrafficLightColor.msg

int32  id      # unique identifier of the traffic light
string color   # detected color: "red", "yellow" or "green"

